config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -319,6 +319,67 @@ periodics:
           cpu: 6
           memory: "16Gi"
 
+- name: ci-kubernetes-kubemark-gce-scale-scheduler
+  tags:
+  - "perfDashPrefix: kubemark-5000Nodes-scheduler"
+  - "perfDashJobType: performance"
+  cron: '1 12 2-31/2 * *' # Run every second day at 12:01 UTC starting on the 2nd of each month
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+    preset-e2e-kubemark-common: "true"
+    preset-e2e-kubemark-gce-scale: "true"
+  annotations:
+    testgrid-dashboards: sig-scalability-kubemark
+    testgrid-tab-name: kubemark-5000-scheduler
+    testgrid-num-failures-to-alert: '1'
+    testgrid-num-columns-recent: '3'
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20200901-eeeadc5-master
+      args:
+      - --repo=k8s.io/kubernetes=master
+      - --repo=k8s.io/perf-tests=master
+      - --root=/go/src
+      - --timeout=1100
+      - --scenario=kubernetes_e2e
+      - --
+      - --cluster=kubemark-5000
```

Comment:
As per https://github.com/kubernetes/perf-tests/pull/1464#discussion_r487918757, I have concerns about whether using the same cluster is a good idea. While we only have plans to have 3 workloads (anti-affinity, affinity, spreading), assuming that each takes at least ~13 hours, I think it might be prudent to:

1. Use a different cluster so that we won't have false positives from the jobs overlapping. While this is a less of a concern with the 100-node job, I think it might be a good idea to do it there as well.
2. Run the 5k job weekly - the scheduler team/sig doesn't need daily data for the 5k job, weekly results for the 5k job are sufficient. There might be more workloads added to the scheduler suite in the future.

cc @ahg-g for opinions on (2)?

Reply:
I think we should aim for at least two runs per week.

Reply:
I asked about this already in the other PR, but I'd like to understand why a single test takes 13h. It seems super long comparing to the vanilla density test which takes less than 2h